src/D2L.Bmx/Okta/OktaClient.cs

```diff
@@ -23,20 +23,21 @@ string challengeResponse
 
 internal interface IOktaAuthenticatedClient {
 	Task<OktaApp[]> GetAwsAccountAppsAsync();
+	Task<OktaSession> GetCurrentOktaSessionAsync();
 	Task<string> GetPageAsync( string url );
 }
 
 internal class OktaClientFactory : IOktaClientFactory {
 	IOktaAnonymousClient IOktaClientFactory.CreateAnonymousClient( string org ) {
```

Comment:
nit: now that you're changing the meaning of the `org` parameter, best to rename it appropriately too, e.g. `orgUrl`
Can also consider changing the type to `Uri` to better enforce this.

Reply:
Okay I tried to move everything to `Uri`